.github/workflows/continuous.yaml

```diff
@@ -266,6 +266,79 @@ jobs:
+    steps:
+      # BEGIN SETUP
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Set Up Python 3
+        with: 
+          python-version: '3.7'  
+        uses: actions/setup-python@v1
+      - name: Set Up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      # Set Up Connection to Dev Mongo
+      # 1. Auth to cluster
+      # 2. Open encrypted connection
+      - uses: google-github-actions/setup-gcloud@v0
+        with:
+          service_account_key: ${{ secrets.DEV_CLOUDBUILD_SA_KEY }}
+          project_id: ${{ secrets.DEV_PROJECT }}
+      - name: Authenticate GHA Runner To Target Cluster
+        run: gcloud container clusters get-credentials ${{secrets.DEV_GKE_CLUSTER}} --zone ${{secrets.DEV_GKE_REGION}} --project ${{secrets.DEV_GCP_PROJECT}}
+      - name: Establish MongoDB Connection
+        run: kubectl port-forward svc/mongo 27017:27017 &
+      - name: Await MongoDB Connection
+        run: sleep 20
+      - name: Confirm MongoDB Connection
+        # The following command returns a success code if mongo is available,
+        # and a failure code if it is not
+        run: curl localhost:27017
+      - name: Establish PostgreSQL Connection
+        run: kubectl port-forward svc/postgres 5432:5432 &
+      - name: Await MongoDB Connection
+        run: sleep 20
+      - name: Confirm PostgreSQL Connection
+        # if the PostgreSQL server is available, curl will fail with a '52' exit code
+        # if it is NOT available, it will fail with a '7' exit code
+        # run: curl localhost:5432; if [ $? -eq "52" ]; then exit 0; else exit 1; fi
+        run: exitCode=$(curl localhost:5432; echo $?); if [ $exitCode -eq 52 ]; then exit 0; else exit 1; fi
+      - name: Set outputs
+        id: get-sha
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+      # END SETUP
+      - name: Install Python Packages
+        run: pip3 install -r ./requirements.txt
+      - name: Install test dependencies
+        run: npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+      - name: Run Playwright tests
+        run: npx playwright test
+        env:
+          CI_RUN: "true"
+          CI_URL: https://sandbox-${{ steps.get-sha.outputs.sha_short }}.cauldron.sefaria.org
+          SEFARIA_TEST_PASS: ${{ secrets.SEFARIA_APP_PASS }}
+          SEFARIA_TEST_USER: ${{ secrets.SEFARIA_APP_USER }}
+          GITHUB_SHA_SORT: ${{ steps.get-sha.outputs.sha_short }}
+      - uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: playwright-report
+          path: playwright-report/
+          retention-days: 30
+      - name: Authenticate GHA Runner to GCP
+        uses: google-github-actions/setup-gcloud@v0
+        with:
+          service_account_key: ${{ secrets.GCS_BUILD_UPLOAD_KEY }}
+          project_id: ${{ secrets.BUILD_UPLOAD_PROJECT }}
+        if: ${{ always() }}
```

Comment:
Rather make sure that the runner JWT has the appropriate set of permissions to perform the requisite actions?  If this pipeline needs a different set of permissions to the regular github actions that you don't want to have stray into other jobs, then we should set up a second WIF binding that is only accessible in this workflow